spec/models/manageiq/providers/amazon/network_manager_spec.rb

```diff
@@ -21,16 +21,6 @@
       expect(described_class.hostname_required?).to eq(false)
     end
 
-    it "returns the expected value for the default_blacklisted_event_names method" do
-      array = %w(
-        ConfigurationSnapshotDeliveryCompleted
-        ConfigurationSnapshotDeliveryStarted
-        ConfigurationSnapshotDeliveryFailed
-      )
-
-      expect(described_class.default_blacklisted_event_names).to eq(array)
-    end
-
```

Comment:
The NetworkManager::EventCatcher was removed a long time ago in 15db06f9a423af55afed7267448b615507703225 and the default_blacklisted_event_names method was removed in 991a51a4c9d4eba63a3694a169007d899c56ac11 so this test is not necessary.